packages/desktop-gui/src/project/project.jsx

```diff
@@ -1,18 +1,49 @@
+import _ from 'lodash'
 import React, { Component } from 'react'
 import { observer } from 'mobx-react'
 import Loader from 'react-loader'
+import Markdown from 'markdown-it'
 
 import C from '../lib/constants'
 import projectsApi from '../projects/projects-api'
 import appStore from '../lib/app-store'
 import viewStore from '../lib/view-store'
+import ipc from '../lib/ipc'
 
 import Settings from '../settings/settings'
 import OnBoarding from './onboarding'
 import ProjectNav from '../project-nav/project-nav'
 import RunsList from '../runs/runs-list'
 import SpecsList from '../specs/specs-list'
 
+const md = new Markdown({
+  html: true,
+  linkify: true,
+})
+
+const ErrorDetails = observer(({ err }) => {
+  let details = _.clone(err.details).split('\n')
```

Comment:
Not seeing the reason this needs to be a `let` instead of `const`

Reply:
Left over from when I was updating `details` later in the file, I'll update. (Still a WIP 😬)